src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -57,13 +58,25 @@ void upgrade() throws IOException {
         Params params = Params.create(upgrade, buildToolStrategy, latestBuildToolVersion, layout.getProjectDirectory(), layout.getBuildDirectory(), gitHub);
 
         if (!prExists(params)) {
-            createPrIfWrapperUpgradeAvailable(params);
+            cloneGitProject(params);
+            VersionInfo usedBuildToolVersion = buildToolStrategy.extractCurrentVersion(params.rootProjectDir);
+            if (isCurrentVersionNewer(usedBuildToolVersion, params.latestBuildToolVersion)) {
+                getLogger().lifecycle(String.format("Project '%s' %s Wrapper current version '%s' is equal or newer than latest version '%s' available",
+                    params.project, buildToolStrategy.buildToolName(), usedBuildToolVersion.version, params.latestBuildToolVersion.version));
```

Comment:
I think we should still try to create a PR in this scenario since it could be that the sha or so is not properly defined. Basically:
 - always try to create a PR
 - use either the usedBuildToolVersion or the latestBuildToolVersion, depending on which one is newer

Reply:
ah forgot about this use case, will rework this on Monday

Reply:
I think in the end, there will be no more code branches here, and the only difference will happen a step earlier when you determine the Gradle/Maven version to create a (potential) PR for: latest version vs. current version.

Reply:
I've made the corresponding change by setting the version when creating the `Params` object. It has the side effect of cloning the project every time, since we need it to know what's the current version.